spyder_terminal/tests/test_terminal.py

```diff
@@ -167,16 +167,17 @@ def register_shortcut(self, *args, **kwargs):
 
     def teardown():
         terminal.on_close()
+        CONF.unregister_plugin(TerminalPlugin)
 
     request.addfinalizer(teardown)
     return terminal
 
 
 @flaky(max_runs=3)
 @pytest.mark.skipif((os.environ.get('CI') and
```

Comment:
```suggestion
@pytest.mark.skipif(
    os.environ.get('CI') and not sys.platform == "darwin",
```

Simplify skip condition.